src/policy/fees.cpp

```diff
@@ -710,6 +711,7 @@ CFeeRate CBlockPolicyEstimator::estimateRawFee(int confTarget, double successThr
 
 unsigned int CBlockPolicyEstimator::HighestTargetTracked(FeeEstimateHorizon horizon) const
 {
+    LOCK(m_cs_fee_estimator);
```

Comment:
I don't think we need exclusive locking here, as `GetMaxConfirms` is a read so concurrent access wouldn't result in incoherent data.

Reply:
@Empact Do you mean that the following annotations are incorrect and should be change or removed?

```
    std::unique_ptr<TxConfirmStats> feeStats GUARDED_BY(m_cs_fee_estimator);
    std::unique_ptr<TxConfirmStats> shortStats GUARDED_BY(m_cs_fee_estimator);
    std::unique_ptr<TxConfirmStats> longStats GUARDED_BY(m_cs_fee_estimator);
```